### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by Yanom1212** [bropat/hassio-eufy-security-ws#1]
### Add-on version

1.3.2

### HomeAssistant version

2023.2.5

### Operating System type

Linux

### Operating system version

Home Assistant Supervisor

### Describe the bug

Échec de l'enregistrement de la configuration du module complémentaire, Invalid list for option 'stations' in eufy-security-ws (402f1039_eufy_security_ws). Got {'username': 'xxxxxx', 'password': 'xxxxxxx', 'country': 'CA', 'port': 3000, 'polling_interval': 10, 'accept_invitations': True, 'debug': False, 'event_duration': 10}

### To reproduce

1. Edit config

### Screenshots & Logfiles

![Capture d’écran 2023-02-23 183620](https://user-images.githubusercontent.com/75578142/221055653-b18be8d0-784f-41f7-9033-bacaf39fef8d.png)

**Comment by bropat** [comment]
As a workaround enter something like this:

```yaml
- serial_number: T8010XXXXXXXXXX
  ip_address: 192.168.0.100
```

I need to see how to make a list parameter optional....

**Comment by WillWNL** [comment]
![Knipsel](https://user-images.githubusercontent.com/32453269/221245008-47c50d88-f5e7-488b-bdf2-b182e7f36d8c.PNG)


Getting the same here on a RPI4

**Comment by bropat** [comment]
@WillWNL 

In your case the format of the `Stations` parameter is incorrectly formatted (see example in the documentation tab).
Your error message is correct and not an error ;)

**Comment by bropat** [comment]
@Yanom1212 
In your case you can insert `[]` if you do not want to configure this parameter.
In the next release this will be the default value.

**Comment by bropat** [comment]
Fixed in release `1.3.3`.
